overlord/devicestate/devicestate_gadget_test.go

```diff
@@ -1170,12 +1195,23 @@ func (s *deviceMgrGadgetSuite) testGadgetCommandlineUpdateRun(c *C, fromFiles, t
 		// we log on success
 		log := tsk.Log()
 		if logMatch != "" {
-			c.Assert(log, HasLen, 1)
+			if !isClassic {
+				c.Assert(log, HasLen, 1)
+			} else {
+				// XXX CLASSIC-NO-REBOOT
```

Comment:
(nitpick^2) I think the intend here is clear, the `XXX` means that the `isClassic=false` will be removed once the reboot delay code lands(?).

Reply:
yes